packages/rocketchat-e2e/client/rocketchat.e2e.room.js

```diff
@@ -285,7 +285,7 @@ export class E2ERoom {
 
 		try {
 			const result = await decryptAES(vector, this.groupSessionKey, cipherText);
-			return EJSON.parse(toString(result));
```

Comment:
Since there is more references to `toString()` in this module, wasn't it better to change it in `helper.js`?